holmes/utils/llms.py

```
@@ -0,0 +1,20 @@
+    Args:
+        model: The name of an LLM model (e.g., "azure/gpt", "openai/gpt-4o")
+        model_list: List of model patterns that may include wildcards
+                   (e.g., ["azure/*", "*/mistral", "openai/gpt-*"])
+
+    Returns:
+        True if the model matches any pattern in the list, False otherwise
+    """
+    for pattern in model_list:
+        if fnmatch.fnmatchcase(model, pattern):
+            return True
+    return False
```

Ruff (0.12.2)

Use `return any(fnmatch.fnmatchcase(model, pattern) for pattern in model_list)` instead of `for` loop

Replace with `return any(fnmatch.fnmatchcase(model, pattern) for pattern in model_list)`

(SIM110)